docs/.vitepress/locales/es.ts

```diff
@@ -0,0 +1,72 @@
+import { DefaultTheme, LocaleConfig } from "vitepress";
+import * as common from "./common";
+export const themeConfig: DefaultTheme.Config = {
+  ...common.themeConfig,
+  editLink: {
+    pattern:
+      "https://github.com/omnivore-app/docs.omnivore/edit/main/docs/:path",
+    text: "Editar esta página en GitHub",
+  },
+  sidebar: [
+    {
+      text: "Usando Omnivore",
+      items: [
+        { text: "Instalación", link: "/using/install" },
```

Comment:
hey the one issue with this is the links should all be like `/es/using/install`. I can merge this though and adjust pretty quickly.